integration-tests/pom.xml

```diff
@@ -208,6 +208,45 @@
     </profile>
 
     <profile>
+      <id>swiftTest</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.tachyonproject</groupId>
+          <artifactId>tachyon-underfs-swift</artifactId>
+          <version>${project.version}</version>
+          <scope>test</scope>
+        </dependency>
+      </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <!--argLine>-XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp/dump</argLine-->
+              <systemPropertyVariables>
+                <ufs>tachyon.underfs.swift.SwiftUnderStorageCluster</ufs>
+                <!-- Change "myAuthUrlKey" to point authentication URL -->
+                <authUrlKey>myAuthUrl</authUrlKey>
+                <!-- Change "myAuthMethodKey" to define authentication type -->
+                <authMethodKey>keystone</authMethodKey>
+                <!-- Change "myUserKey" to contain the user  -->
+                <userKey>myUserKey</userKey>
+                <!-- Change "myApiKey" to contain the user's key or password  -->
+                <apiKey>myApiKey</apiKey>
+                <!-- Change "myTenantKey" to contain the user's tenant or account  -->
+                <tenantKey>myTenantKey</tenantKey>
+                <!-- Change "myContainerKey" to contain the testing container  -->
+		<!-- In a form container/folder -->
+                <containerKey>swift://<container/folder></containerKey>
```

Comment:
I think the `<container/folder>` is messing up the pom parsing, could you replace the delimiters with something other than `<` and `>`?